es-ds-styles/scss/_dropdown.scss

```diff
@@ -201,3 +201,137 @@
   padding: variables.$dropdown-item-padding-y variables.$dropdown-item-padding-x;
   color: variables.$dropdown-link-color;
 }
+
+// ==========================================
+// ES-DROPDOWN CUSTOM STYLES
+// ==========================================
+
+// Custom es-dropdown input styling
+.es-dropdown-input {
+  align-items: center;
+  background-color: variables.$input-bg;
+  background-clip: padding-box;
+  border: 1px solid variables.$gray-500;
+  border-radius: variables.$border-radius-xs;
+  color: variables.$input-color;
+  display: flex;
+  height: variables.$input-btn-height;
+  justify-content: space-between;
+  padding: 16px 16px 16px 32px;
+  font-family: variables.$input-font-family;
+  font-size: variables.$input-font-size;
+  font-weight: variables.$input-font-weight;
+  line-height: variables.$input-line-height;
+  text-decoration: none;
+  transition: border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+  white-space: nowrap;
+  width: 325px;
```

Comment:
Our dropdowns are definitely going to need to vary in width based on the use case and the text inside them, so I don't think we should hardcode a width here. Dropdowns should just always take up the full width of their container and we can let the downstream app authors limit the width as necessary for their use case.